dbos/cli/cli.py

```diff
@@ -363,5 +358,61 @@ def restart(
         print(f"Failed to resume workflow {uuid}. Status code: {response.status_code}")
 
 
+@queue.command(name="list", help="List enqueued functions for your application")
+def list_queue(
+    limit: Annotated[
+        typing.Optional[int],
+        typer.Option("--limit", "-l", help="Limit the results returned"),
+    ] = None,
+    start_time: Annotated[
+        typing.Optional[str],
+        typer.Option(
+            "--start-time",
+            "-s",
+            help="Retrieve functions starting after this timestamp (ISO 8601 format)",
+        ),
+    ] = None,
+    end_time: Annotated[
+        typing.Optional[str],
+        typer.Option(
+            "--end-time",
+            "-e",
+            help="Retrieve functions starting before this timestamp (ISO 8601 format)",
+        ),
+    ] = None,
+    status: Annotated[
+        typing.Optional[str],
+        typer.Option(
+            "--status",
+            "-S",
+            help="Retrieve functions with this status (PENDING, SUCCESS, ERROR, RETRIES_EXCEEDED, ENQUEUED, or CANCELLED)",
+        ),
+    ] = None,
+    queue_name: Annotated[
+        typing.Optional[str],
+        typer.Option(
+            "--queue-name",
+            "-q",
+            help="Retrieve functions on this queue",
```

Comment:
```suggestion
            help="Retrieve tasks on this queue",
```

or "workflows" ?

Reply:
In our docs we always call them functions.